apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -776,6 +802,27 @@ void wm_get_stats(void *arg)
 	WM_TEST_LOG_END;
 }
 
+void wm_get_conn_info(void *arg)
+{
+	WM_TEST_LOG_START;
+	wifi_manager_info_s info;
+	wifi_manager_result_e res = wifi_manager_get_info(&info);
+	if (res != WIFI_MANAGER_SUCCESS) {
+		printf("[WM] fail to get connection info(%d)\n", res);
```

Comment:
It means we cannot get the info well. But in the below, we are trying to print the info. I think they will be useless in fail case.